### PR TITLE
THEVOICE-6 migrate from tslint to eslint

### PR DESCRIPTION
**Pull request by henckellbach**
It's time to move away from TSLint _(deprecated)_ to ESLint and revive this repo!

### CODE REVIEW

File: common/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  ignorePatterns: ['node_modules/*', '/codegen/*', '/public/*', '/tests/**'],
+  rules: {
+    'react/no-unescaped-entities': ['error', { forbid: ['>', '}'] }], // Allow quote marks in text
+    'react/prop-types': 'off', // Allow not providing PropTypes as we use FC<Props> to type props
+    'react/react-in-jsx-scope': 'off', // Allow not importing React as Next imports it automatically
```
Comment (Ketcap) [inline common/.eslintrc.js:26]: shouldn't this override on the project-specific base?

**Comment by alizhdanov** [comment]
Looks good. The only thing I'd keep more generic config in the `common` folder and in the `frontend` folder extend it and add all frontend related things.

File: frontend/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2020,
+    sourceType: 'module',
+  },
+  extends: [
```
Comment (alizhdanov) [inline frontend/.eslintrc.js:10]: I'd extend from common config and remove duplicity. Check my [PR](https://github.com/oakslab/ooo-config-examples/pull/8/files#diff-552c24c518fbc7b1401230074f9350f29c3cce7744ae9431173f2dc16d81f745R8)

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@oakslab/ooo-config",
-  "version": "0.0.7",
+  "version": "0.0.8",
   "description": "Shared configuration files for OAK'S LAB projects",
   "license": "MIT",
   "author": "OAK'S LAB",
```
Comment (alizhdanov) [inline package.json:6]: I think you forgot to install dependencies 🤔 E.g. `typescript-eslint`.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@oakslab/ooo-config",
-  "version": "0.0.7",
+  "version": "0.0.8",
```
Comment (alizhdanov) [inline package.json:3]: Let's make a minor release (0.1.0) as it has some breaking changes.

**Review by alizhdanov — CHANGES_REQUESTED** [review]
Almost perfect 👍

**Review by alizhdanov — APPROVED** [review]

**Pull request merged**